services/movie_session.py

```diff
@@ -42,3 +42,12 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    tickets = Ticket.objects.filter(movie_session__id=movie_session_id)
```

Comment:
`__id` lookup is not necessary here; use `_id`. `ticket` table already contains it.

Reply:
@mltdll btw, as I know, Danylo told that it is better to use `__id` because sometimes we can create a field with a `_id` suffix and an error may occur. But I use `_id` as well 😅

Reply:
Got it. Thanks for the information!
It seems that it would be even more flexible to use `__pk` lookup because that way it doesn't depend on the name of the primary key field of the other table.